### ISSUE TITLE
Tries to set auto_increment with thousand separator

### ISSUE THREAD

**Issue by dmoagx** [sequelpro/sequelpro#2130]
When using the "Set auto increment to..." feature to a value > 1000 Sequel Pro somehow tries to write the value including the thousand separator:
```sql
/* 21:08:32.651 sequelpro_log */ ALTER TABLE `reports` AUTO_INCREMENT = 98.062;
```
(note this is a german locale system, so `,` is the decimal seperator, `.` is the thousand seperator)
This results in MySQL interpreting the value as a decimal and using `98` instead.

**Comment by abhibeckert** [comment]
We need to merge this into the 1.1 branch as well.

**Comment by dmoagx** [comment]
Merged to the 1.1 branch.
